autopilot/betweenness_centrality.go

```diff
@@ -0,0 +1,177 @@
+package autopilot
+
+// stack is a simple int stack to help with readability of Brandes'
+// betweenness centrality implementation below.
+type stack struct {
+	stack []int
+}
+
+func (s *stack) Push(v int) {
```

Comment:
exported functions should have godoc?

Reply:
we really don't need to export them so instead of adding the trivial godoc, made them local to this file instead.